src/sisl_toolbox/siesta/minimizer/_minimize.py

```diff
@@ -54,12 +57,15 @@ class BaseMinimize:
     # Basic minimizer basically used for figuring out whether
     # to use a local or global minimization strategy
 
-    def __init__(self, variables=(), out="minimize.dat", norm="identity"):
+    def __init__(
+        self, variables=(), out="minimize.dat", norm="identity", out_fmt="20.17e"
```

Comment:
should we rename this to `fmt`, isn't it clear enough? This is what we do for other siles?

Reply:
hmm, I see. :)